### DISCUSSION TITLE
Will it support reviewing notebooks stored on s3?

### DISCUSSION THREAD

**Discussion by jxudata** [ReviewNB/support#118] [Ideas]
Some notebooks are stored on s3 (accessed by JupyterHub) and I am wondering if it's possible to review s3 notebooks. Thanks!

**Comment by amit1rrr** [comment]
S3 notebooks are not supported at the moment but we'd like to understand a little bit more about the use case - 

- What type of notebooks are these & who would like to review? What is the goal of these reviews?
- Do you care about diff between various versions of the notebook or just want to review single notebook as-is?
- Have you considered moving these S3 notebooks to a git repository on GitHub / bitbucket? Are there any challenges / preferences here?
- How do you imagine access control would look like? I.e. who has access to which S3 notebook etc.
